code/controllers/subsystem/sounds.dm

```diff
@@ -126,6 +126,9 @@ SUBSYSTEM_DEF(sounds)
 /datum/controller/subsystem/sounds/proc/random_available_channel()
 	if(channel_random_low > channel_reserve_high)
 		channel_random_low = 1
+	if(channel_random_low++ > length(channel_list))
+		. = channel_list[length(channel_list)]
+		return
```

Comment:
I'm not actually sure if this is right, it'll certainly fix the runtime, the question is, is it the right thing to do? I don't know the subsystem well enough to know.